lib/engine.py

```diff
@@ -426,12 +419,12 @@ def download_files_from_archive(self, url, filenames, filetype="zip",
 
                 if filetype == 'zip':
                     archive = zipfile.ZipFile(archivename)
-                    open_archive_file = archive.open(filename)
+                    open_archive_file = archive.open(filename, 'rU')
```

Comment:
This looks fine to me. but before changing this. i would recommend dealing with `def download_file(self, url, filename, clean_line_endings=True):`